packages/nimble-blazor/NimbleBlazor/Components/NimbleWaferMap.razor.cs

```diff
@@ -0,0 +1,179 @@
+﻿using System.Text.Json;
+using Microsoft.AspNetCore.Components;
+using Microsoft.JSInterop;
+
+namespace NimbleBlazor;
+
+/// <summary>
+/// A visualization widget which displays the location and color code of integrated circuits on a silicon wafer.
+/// </summary>
+public partial class NimbleWaferMap : ComponentBase
+{
+    private ElementReference _waferMap;
+    private bool _diesUpdated = false;
+    private IEnumerable<WaferMapDie>? _dies = Enumerable.Empty<WaferMapDie>();
+    private bool _colorScaleUpdated = false;
+    private WaferMapColorScale? _colorScale;
+    private bool _highlightedTagsUpdated = false;
+    private IEnumerable<string>? _highlightedTags = Enumerable.Empty<string>();
+    internal static string GetWaferMapValidityMethodName = "NimbleBlazor.WaferMap.getValidity";
+    internal static string SetWaferMapDiesMethodName = "NimbleBlazor.WaferMap.setDies";
+    internal static string SetWaferMapColorScaleMethodName = "NimbleBlazor.WaferMap.setColorScale";
+    internal static string SetWaferMapHighlightedTagsMethodName = "NimbleBlazor.WaferMap.setHighlightedTags";
+
+    [Inject]
+    private IJSRuntime? JSRuntime { get; set; }
+
+    /// <summary>
+    /// Represents the starting point and the direction of the two axes, X and Y, which are used for displaying the die grid on the wafer map canvas.
+    /// </summary>
+    [Parameter]
+    public WaferMapOriginLocation? OriginLocation { get; set; }
+
+    /// <summary>
+    /// Represents the X coordinate of the minimum corner of the the grid bounding box for rendering the wafer map.
+    /// </summary>
+    [Parameter]
+    public double? GridMinX { get; set; }
+
+    /// <summary>
+    /// Represents the Y coordinate of the minimum corner of the the grid bounding box for rendering the wafer map.
```

Comment:
Are these docs accurate / are they reversed with those of `GridMinY`? (right now the docs for `GridMaxX` say `Y coordinate of the minimum corner`)